specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -0,0 +1,255 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-11-01-preview",
+    "title": "BillingManagementClient",
```

Comment:
normally when a new version of an API is created the existing resources/operations are exposed from the most recent preview are also included and not only the new stuff.  Is it intended to not include content from https://github.com/Azure/azure-rest-api-specs/blob/master/specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

Reply:
Agreed.. Currently with this new version we have spec ready for just resourceprices. Rest all stuff we are still working on finalizing contract.